TerminalDocs/customize-settings/actions.md

```diff
@@ -858,26 +858,6 @@ You can mark a pane as read-only, which will prevent input from going into the t
 { "command": "toggleReadOnlyMode" }
 ```
 
-You can enable read-only mode on a pane. This works similarly to toggling, however, will not switch state if triggered again.
-
-**Command name:** `enableReadOnlyMode`
-
-**Default bindings:**
-
-```json
-{ "command": "enableReadOnlyMode" }
-```
-
-You can disable read-only mode on a pane. This works similarly to toggling, however, will not switch state if triggered again.
-
-**Command name:** `disableReadOnlyMode`
-
-**Default bindings:**
-
-```json
-{ "command": "disableReadOnlyMode" }
-```
-
```

Comment:
Looks unrelated?

Reply:
Good catch. I removed this in the latest commit.